### ISSUE TITLE
Low Semantic Segmentation Performance

### ISSUE THREAD

**Issue by brucemuller** [mapillary/inplace_abn#110]
Hi Mapillary

I'm trying to implement your Semantic Segmentation model from your InPlaceABN paper. So far my results achieve around 8% mean IoU on the validation set. I don't want to clog this post with the code but I'm happy to share if requested. I'm using a batch size of 150 and a fixed SGD LR of 0.001. I've attached some example results and training curves. Do you have any idea on what's likely the problem?

I'd like to know how you trained your state of the art segmentation model. Did you use class balancing in the loss layer or any pre-training for this task? I'd like to get this working so then I can cite your paper in my upcoming publication!

![abn_vistas_epoch200_ex6](https://user-images.githubusercontent.com/48244902/55405783-b2fb3e80-5552-11e9-9d60-112069e75f5f.png)
![abn_vistas_epoch200_ex7](https://user-images.githubusercontent.com/48244902/55405784-b2fb3e80-5552-11e9-8df6-cde705430305.png)
![abn_vistas_epoch200_ex8](https://user-images.githubusercontent.com/48244902/55405785-b2fb3e80-5552-11e9-80d6-b23a2fb700d4.png)
![abn_vistas_epoch200_ex9](https://user-images.githubusercontent.com/48244902/55405786-b393d500-5552-11e9-9dc4-b61c9c3af7f6.png)
![abn_vistas_epoch200_ex10](https://user-images.githubusercontent.com/48244902/55405787-b393d500-5552-11e9-8906-46d0201972e7.png)
[abn_vistas_IoU.pdf](https://github.com/mapillary/inplace_abn/files/3034104/abn_vistas_IoU.pdf)
[abn_vistas_loss.pdf](https://github.com/mapillary/inplace_abn/files/3034105/abn_vistas_loss.pdf)

**Comment by rotabulo** [comment]
@brucemuller247 we provide details about our model and how we trained it in the paper https://arxiv.org/abs/1712.02616

**Comment by ducksoup** [comment]
In an effort to streamline the resolution process we are closing any issue not directly related with the InPlace-ABN _library and scripts_. Please use other channels if you need to contact us about our paper or the Mapillary Vistas dataset.
